sql/conseil.sql

```diff
@@ -869,3 +869,69 @@ FROM
     AND bitcoin.outputs.n = bitcoin.inputs.v_out
   GROUP BY
     bitcoin.outputs.script_pub_key_addresses;
+
+CREATE SCHEMA ethereum; 
+
+-- List of the Ethereum based blockchains implemented in Conseil
+CREATE TYPE ethereum.networks AS ENUM ('ethereum', 'quorum');
```

Comment:
Food for thought: Will it work fine like this, or will it hinder code flexibility to "encode" this choice at the DB level with no explicit mapping at the code level?

Reply:
I also have a problem with this line, I recently used `rinkeby` network for ETH and I believe it should be included in metadata conf, not hardcoded in the DB. Other reason is that, I don't know how often new test network appears, so we would need to add new network to the schema.

Reply:
By the way, it's not going to be `ethereum` and `quorum` rather `mainnet` (for Ethereum) and `quorum` (for JP Morgan Quorum). If they have different networks, then it should be named `quorum-xyz` I believe. Note that the url for this endpoint will be: `/data/ethereum/mainnet/blocks` or `/data/ethereum/quorum/blocks`. If we want to have `/data/quorum/mainnet/blocks` for the latter example, then we are not talking about networks, but about platforms and it makes entire configuration a bit more complicated, because this part is not "dynamic" in current design. We agreed to treat `Quorum` as `Ethereum`. As you can see, this part is quite dynamic and actually API is going to handle the case, when given network does not exists in the database, so I'm not sure either if we should limit this here. After some consideration maybe we should remove that in initial commits?

Reply:
I've removed network column